draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -253,6 +250,20 @@ The schemes operations defined in this section depend on the following parameter
 
 * get\_random(n): returns a random octet string with a length of n bytes, sampled uniformly at random using a cryptographically secure pseudo-random number generator (CSPRNG) or a pseudo random function. See [@!RFC4086] for recommendations and requirements on the generation of random numbers.
 
+## Interfaces
+
+The BBS signature scheme is organized as follows:
+
+- A set of low level (core) operations, taking care of the main cryptographic functionality.
+- An Application Interface (API), that uses the core operations in a secure way.
+
+Each of the core operations (see (#core-operations)), expect a list of points (called the generators, see (#generators)) and a list of messages represented as scalar values (see (#messages)). It is the job of the Interface to:
+
+1. Create the necessary generators.
+2. Map the inputted messages to scalars.
```

Comment:
is there a better word here than inputted? are these messages named in the spec? can we include a ref here?